frontend/dockerfile/dockerfile2llb/convert.go

```diff
@@ -243,7 +252,7 @@ func Dockerfile2LLB(ctx context.Context, dt []byte, opt ConvertOpt) (*llb.State,
 					}
 					d.stage.BaseName = reference.TagNameOnly(ref).String()
 					var isScratch bool
-					if metaResolver != nil && reachable && !d.unregistered {
+					if metaResolver != nil && reachable {
```

Comment:
Copy from image will now load image config here although it is not needed to get an error in the correct place. The config would have been still downloaded together with the image layers before.